### PR TITLE
adding required env vars

### PR DESCRIPTION
**Pull request by noakup**

### CODE REVIEW

File: cloudify_agent/installer/__init__.py [hunk 1]
```diff
@@ -198,14 +198,14 @@ def _create_agent_env(self):
...
             env.CLOUDIFY_BROKER_PORT: self.cloudify_agent.get('broker_port'),
-            env.CLOUDIFY_FILE_SERVER_PORT:
-                self.cloudify_agent['file_server_port'],
-            env.CLOUDIFY_FILE_SERVER_PROTOCOL:
-                self.cloudify_agent.get('file_server_protocol'),
             env.CLOUDIFY_REST_PORT:
                 self.cloudify_agent.get('rest_port'),
             env.CLOUDIFY_REST_PROTOCOL: self.cloudify_agent.get(
                 'rest_protocol'),
+            env.CLOUDIFY_FILE_SERVER_PORT:
```
Comment (dankilman) [inline cloudify_agent/installer/__init__.py]: why is this changed to get?

File: cloudify_agent/installer/config/configuration.py [hunk 2]
```diff
@@ -164,7 +164,6 @@ def _cfy_agent_attributes_no_defaults(cloudify_agent):
             cloudify_utils.get_manager_file_server_host()
 
     if not cloudify_agent.get('file_server_port'):
-        # by default, file_server_port is set by an environment variable
```
Comment (dankilman) [inline cloudify_agent/installer/config/configuration.py]: why is this comment removed?

File: system_tests/manager/test_agent_installer.py [hunk 3]
```diff
@@ -232,22 +234,41 @@ def _manager_host(self):
...
             'rest_host': manager_host,
+            'file_server_host': manager_host,
+            'broker_ip': manager_host,
             'name': name
         }})
     try:
         current_ctx.set(ctx)
-        os.environ['MANAGER_FILE_SERVER_URL'] = 'http://{0}:53229'.format(
-            manager_host)
+        for var_name, value in env_vars.iteritems():
```
Comment (dankilman) [inline system_tests/manager/test_agent_installer.py]: this can actually be simplified to `os.environ.update(env)`

File: cloudify_agent/installer/config/configuration.py (hunk 2 above)
Comment (noakup) [inline cloudify_agent/installer/config/configuration.py]: simple because it's true for every other var as well, and pretty clear from the code as it is

File: cloudify_agent/installer/__init__.py (hunk 1 above)
Comment (noakup) [inline cloudify_agent/installer/__init__.py]: since it's the same (in terms of being mandatory or not) as the rest_port and protocol
